book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
Since we will be making modifications to the rental class, I suggest you add getters and setters so we can read and write into it😉.

```suggestion
  attr_accessor :title, :author, :rentals
```